### PR TITLE
Update scripts

### PR DESCRIPTION
**Pull request by TotallyNotElite**
- Install git independently of other packages (Stable may have other dependencies than master. Also allows us to remove some package install hacks from nullworks/One-in-all-cathook-install)
- Improve the update scripts capabilities to recover a bad repo

### CODE REVIEW

File: install-all [hunk 1]
```diff
@@ -5,23 +5,6 @@ RUNCMD="sudo -u $RUNUSER"
 
 set -e
 
-# install all dependencies
```
Comment (BenCat07) [inline install-all]: *confused screaming*

Comment (TotallyNotElite) [reply install-all]: ?

Comment (BenCat07) [reply install-all]: why is this gone entirely

Comment (TotallyNotElite) [reply install-all]: https://github.com/nullworks/cathook/pull/861/files#diff-5d98e6a5f9ed1ef89f93d94567219d9cR34
https://github.com/nullworks/cathook/pull/861/files#diff-f72b85a6905939b077ec7953f9594ad7R132

**Pull request merged**